aspnetcore/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps.md

```diff
@@ -0,0 +1,320 @@
+1. On the left navigation, select **Service connector**
+1. On the Service Connector page, choose **+ Create** to open the **Creation Connection** flyout panel and enter the following values:
+    * **Container**: Select the Container App you created previously.
+    * **Service type**: Choose **Storage - blob**.
+    * **Subscription**: Select the subscription you used previously.
+    * **Connection name**: Leave the default value or enter a name of your choosing.
+    * **Storage account**: Select the storage account you created earlier.
+    * **Client type**: Select **.NET**.
+1. Select **Next: Authentication** to progress to the next step.
+1. Select **System assigned managed identity** and choose **Next: Networking**.
+1. Leave the default networking options selected, and then choose **Review + Create**.
+1. After Azure validates your settings, select **Create**.
+
+The service connector enables a system-assigned managed identity on the container app. It also assigns a role of **Storage Blob Data Contributor** to the identity so it can perform data operations on the storage containers.
+
+#### Connect the key vault
+
+1. In the Azure portal, navigate to your Container App overview page.
+1. On the left navigation, select **Service connector**.
+1. On the Service Connector page, choose **+ Create** to open the **Creation Connection** flyout panel and enter the following values:
+    * **Container**: Select the container app you created previously.
+    * **Service type**: Choose **Key Vault**.
+    * **Subscription**: Select the subscription you used previously.
+    * **Connection name**: Leave the default value or enter a name of your choosing.
+    * **Key vault**: Select the key vault you created earlier.
+    * **Client type**: Select **.NET**.
+1. Select **Next: Authentication** to progress to the next step.
+1. Select **System assigned managed identity** and choose **Next: Networking**.
+1. Leave the default networking options selected, and then choose **Review + Create**.
+1. After Azure validates your settings, select **Create**.
+
+The service connector assigns a role to the identity so it can perform data operations on the key vault keys.
+
+## Configure and redeploy the app
+
+The necessary Azure resources have been created, so you'll need to configure your app code to point to those services.
+
+1. Install the following NuGet packages:
+
+    * **Azure.Identity**: Provides classes to work with the Azure identity and access management services.
+    * **Microsoft.Extensions.Azure**: Provides helpful extension methods to perform core Azure configurations.
+    * **Azure.Extensions.AspNetCore.DataProtection.Blobs**: Allows storing ASP.NET Core DataProtection keys in Azure Blob Storage so that keys can be shared across several instances of a web app.
+    * **Azure.Extensions.AspNetCore.DataProtection.Keys**: Enables protecting keys at rest using the Azure Key Vault Key Encryption/Wrapping feature.
+
+    ```dotnetcli
```

Comment:
Normally we tell them to do this in VS, but this is probably OK. See [this sample](https://learn.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-6.0&tabs=visual-studio#scaffold-the-movie-model) if you want to use VS instructions in the future.